packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/temp/LogoAPI.js

```diff
@@ -5,15 +5,27 @@
 
 class LogoAPI {
   constructor() {
-    // this.menuLogo = {
-    //   url: CatLogo,
-    //   name: 'cat-logo.png',
+    // this.projectSettings = {
+    //   menuLogo: {
+    //     url: CatLogo,
+    //     name: 'cat-logo.png',
+    //     width: 100,
+    //     height: 100,
+    //     ext: "png",
+    //     size: 123482,
+    //   }
```

Comment:
Still need to comment this part?

Reply:
I need it until the back-end is ready to handle logo uploads!

Reply:
it allows me to test what would happen with data initialized by data coming from the db